packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -253,34 +255,37 @@ export class DocumentDeltaConnection
     /**
      * Disconnect from the websocket, and permanently disable this DocumentDeltaConnection.
      */
-    public close() {
+    public dispose() {
         this.closeCore(
             false, // socketProtocolError
             createGenericNetworkError("client closing connection", true /* canRetry */));
     }
 
+    // back-compat: became @deprecated in 0.45 / driver-definitions 0.40
+    public close() { this.dispose(); }
+
     protected closeCore(socketProtocolError: boolean, err: DriverError) {
```

Comment:
Should we rename it to disposeCore() now?

Reply:
sure, why not? :)